src/get-default-printer/get-default-printer.ts

```diff
@@ -2,18 +2,17 @@ import execFileAsync from "../utils/exec-file-async";
 import throwIfUnsupportedOperatingSystem from "../utils/throw-if-unsupported-os";
 import isValidPrinter from "../utils/windows-printer-valid";
 
```

Comment:
could you please remove whitespace between these import statements?

Reply:
done